packages/@aws-cdk/aws-batch-alpha/test/ecs-job-definition.test.ts

```diff
@@ -116,3 +116,15 @@ test('JobDefinitionName is parsed from arn', () => {
     ],
   });
 });
+
+test('JobDefinitionName is parsed from arn in imported job', () => {
+  // GIVEN
+  const stack = new Stack();
+
+  // WHEN
+  const importedJob = EcsJobDefinition.fromJobDefinitionArn(stack, 'importedJobDefinition',
+    'arn:aws:batch:us-east-1:123456789012:job-definition/job-def-name:1');
+
+  // THEN
+  expect(importedJob.jobDefinitionName).toEqual('job-def-name:1');
```

Comment:
Ah I think this is an issue. The job definition name should not include the revision.

https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsbatch.html#awsbatch-resources-for-iam-policies

`arn:${Partition}:batch:${Region}:${Account}:job-definition/${JobDefinitionName}:${Revision}`

Reply:
Thanks for the feedback.
Not sure if `split`ting is the best approach here (can `JobDefinitionName` or `Revision` contain `:` characters?).
Anyway, open to suggestions.